client/src/components/Header.js

```diff
@@ -8,7 +8,7 @@ import Logo from "./logonobg.png";
 
 const Header = () => {
   return (
-    <div className="ui menu inverted">
+    <div className="ui menu inverted" style={{ margin: "0px 0px 2px" }}>
```

Comment:
Hmmm is there that tiny gap from before because the header has a bottom margin of 2? Is there a reason you added this margin instead of making everything 0?

Reply:
It looked weird on my monitor when it was rendered. Maybe it's just me.